Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerToContextConnection.cpp

```diff
@@ -210,15 +210,15 @@ void WebSWServerToContextConnection::fireBackgroundFetchEvent(ServiceWorkerIdent
 void WebSWServerToContextConnection::fireBackgroundFetchClickEvent(ServiceWorkerIdentifier serviceWorkerIdentifier, const BackgroundFetchInformation& info, CompletionHandler<void(bool)>&& callback)
 {
     if (!m_processingFunctionalEventCount++)
-        m_connection->networkProcess().parentProcessConnection()->send(Messages::NetworkProcessProxy::StartServiceWorkerBackgroundProcessing { webProcessIdentifier() }, 0);
+        protectedConnection()->protectedNetworkProcess()->protectedParentProcessConnection()->send(Messages::NetworkProcessProxy::StartServiceWorkerBackgroundProcessing { webProcessIdentifier() }, 0);
```

Comment:
These are lot of protected names here, which is hurting a bit readability.
Is the plan to remove the unsafe getters so that we can drop `protected` names on the safe getters?

Reply:
> Is the plan to remove the unsafe getters so that we can drop protected names on the safe getters?
No, as commented the last time you suggested this, we still need the getter that return raw pointers to avoid unnecessary ref-counting when calling trivial member functions.